R/wilcoxonsr_hodges_lehman.qmd

```diff
@@ -0,0 +1,118 @@
+---
+title: "Wilcoxon signed rank test"
+---
+
+### **Introduction**
+
+Wilcoxon signed-rank test is a non-parametric equivalent of paired Student's t-test. It is a rank test, designed for analyzing repeated measures or paired observations by a paired comparison (a type of location test) to assess whether their population means differ. *Ties* are when you have two observations with the same result. For example, in a 2-period cross-over study, you take the difference between result on Treatment A minus result on Treatment B and find that two or more subjects have the same difference.
+
+```{r}
+#| echo: false
+#| include: false
+library(tidyverse)
+library(broom)
+library(knitr)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(stats)
+# library(coin)
+
+renv::restore()
+install.packages("DOS2")
+renv::update()
```

Comment:
This does not need to be included. Basically you have resorted it we don't need to re-restore every time we update the repo and it will cause some issues with the actions to have this
```suggestion

```